package excep;

public class throw1 {

	public static void main(String[] args) {
		if(3>5)
		{
			throw new ArithmeticException("Error");
			
		}

	}

}
